src/TTextEdit.cpp

```diff
@@ -1428,23 +1427,26 @@ QString TTextEdit::getSelectedText(const QChar& newlineChar)
     // mPB QPoint where selection ended
 
```

Comment:
```suggestion
    // try to prevent crash if buffer is batch deleted
    if (mPA.y() > mpBuffer->lineBuffer.size() - 1 || mPB.y() > mpBuffer->lineBuffer.size() - 1){
        mPA.ry() -= mpBuffer->mBatchDeleteSize;
        mPB.ry() -= mpBuffer->mBatchDeleteSize;
    }
```

No idea if this is a good or bad solution (I'm sure you know) but it prevents crashes as I described at https://github.com/Mudlet/Mudlet/issues/3999#issuecomment-692311438 without altering the copied content.